package.json

```diff
@@ -57,6 +57,8 @@
     "test-disablehttp": "REQUIRE_HTTPS=true npm run test-with-coverage -- --recursive tests/disableHttp",
     "test-extra": "npm run test-api-log && npm run test-cache && npm run test-jobQueue && npm run test-view && npm run test-disablehttp && npm run test-cors && npm run test-limiter && npm run test-xfwd && npm run test-routes-enabled && npm run test-routes-disabled && npm run test-realtime-application",
     "test-jobQueue": "ENABLE_BULL_FOR_BULK_DEL_SUBJ=false npm run test-with-coverage -- --recursive tests/jobQueue && ENABLE_BULL_FOR_BULK_DEL_SUBJ=true npm run test-with-coverage -- --recursive tests/jobQueue ",
+    "test-postEventsBulk": "ENABLE_BULL_FOR_BULK_POST_EVENTS=false npm run test-with-coverage -- --recursive tests/api/v1/events && ENABLE_BULL_FOR_BULK_POST_EVENTS=true npm run test-with-coverage -- --recursive tests/api/v1/events ",
+    "test-createAudit": "ENABLE_BULL_FOR_CREATE_AUDIT_EVENTS=false npm run test-with-coverage -- --recursive tests/api/v1/auditEvents && ENABLE_BULL_FOR_CREATE_AUDIT_EVENTS=true npm run test-with-coverage -- --recursive tests/api/v1/auditEvents ",
```

Comment:
These should be included in the existing test-jobQueue script: set all three toggles, then run all the tests at once.